soc/arm/nordic_nrf/nrf53/Kconfig.soc

```diff
@@ -129,6 +129,16 @@ config SOC_DCDC_NRF53X_HV
 	help
 	  Enable nRF53 series System on Chip High Voltage DC/DC converter.
 
+config CONFIG_SOC_ENABLE_LFXO
```

Comment:
Just wondering, if it would be better to add the !NONSECURE and 5340_CPUAPP (i think the latter already is) dependencies at the Kconfig symbol, directly, since this is needed only for nrf5340.

Or do you project that this should be needed for other SoCS?